### DISCUSSION TITLE
MEX file  the specified program could not be found.

### DISCUSSION THREAD

**Discussion by COCOPOLLY** [astra-toolbox/astra-toolbox#517] [Q&A]
When I run the test code ‘astra_test’ , MATLAB gives the following error
MEX file 'D:\Date\MATLAB\Tools by
ChenSun\CSET-master\ASTRA-toolbox\mex\astra_mex_c.mexw64' is invalid: the specified program could not be found.

**Comment by askorikov** [comment]
Hi! What version of Astra do you use? Does installing latest Visual C++ Redistributable (https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170) help?

**Reply by COCOPOLLY** [reply]
Thank you very much for your help, I switched back to version 2.0 and found the problem solved!

**Reply by askorikov** [reply]
Great that the previous version works, but I expect the latest version to work as well. I'm going to look into what's going on. Did you have to install VC redistributable after all?

**Reply by COCOPOLLY** [reply]
I guess it could be a conflict between the versions of cuda, VS and Matlab, I reinstalled vs 2019 cudav11.8 , and found that Astra v2.2 compiles MEX properly!
